Gemfile.lock

```diff
@@ -660,4 +660,4 @@ RUBY VERSION
    ruby 2.4.4p296
 
 BUNDLED WITH
-   1.17.3
+   2.1.0
```

Comment:
Hey @sharmamansi!
I think the test are failing because you edited a line of Gemfile.lock file.
Could you please add a commit to your pull request changing bundler version back to ```1.17.3``` from the version you had changed it to?

I think that should work. Thanks :smile:

Reply:
@Uzay-G Thanks for your help. Please review it again.